src/app_service/service/token/service.nim

```diff
@@ -51,11 +52,21 @@ QtObject:
     events: EventEmitter
     threadpool: ThreadPool
     networkService: network_service.Service
+
+    # TODO: remove these once community usage of this service is removed etc...
     tokens: Table[int, seq[TokenDto]]
     tokenList: seq[TokenDto]
     tokensToAddressesMap: Table[string, TokenData]
+
     priceCache: TimedCache[float64]
 
+    sourcesOfTokensList: seq[SupportedSourcesItem]
+    flatTokenList: seq[TokenItem]
+    tokenBySymbolList: seq[TokenBySymbolItem]
+    # TODO: Table[symbol, TokenDetails] fetched from cryptocompare
```

Comment:
It'd worth it to have a task already created, if it's not!

Reply:
https://github.com/status-im/status-desktop/issues/12668